ProductivitySuite/docker/ui/docker/Dockerfile.react

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Use node 20.11.1 as the base image
+FROM node:20.11.1 as vite-app
+ 
+COPY ./react /usr/app/react
+WORKDIR /usr/app/react
+
+
+RUN ["npm", "install"]
+RUN ["npm", "run", "build"]
+
+
+FROM nginx:alpine
```

Comment:
Have you put this into the BoM list?

Reply:
This should be included previously when we implemented react ui for other examples. @yogeshmpandey  can confirm on this?

Reply:
Added to the BoM list